api/proto/config/storage.proto

```diff
@@ -242,3 +242,71 @@ message DisksConfig {
   DisksArrayType array_type = 2;
   repeated DisksConfig children = 3;
 }
+
+// RollbackCmd - snapshot rollback command
+//
+// If the counter in the DeviceOpsCmd is different, it will cause a
+// rollback operation.
+//
+// The fields from RollbackCmd (snapshot_uuid, volume_uuid) will
+// always match the fields from the SnapshotConfig (snapshot_uuid, volume_uuid)
+// to which RollbackCmd belongs.
+//
+// For example: take the situation where the controller requests one or more
+// snapshots (uuid=1 for volume_uuid=X, then add a second SnapshotCmd
+// with uuid=2 for volume_uuid=X), and then later it send a RollbackCmd with
+// snapshot_uuid=1
+//
+// In this case, the following will happen:
+// It is assumed that the RollbackCmd message cannot come separately from
+// the configuration, and will always be part of the SnapshotConfig for
+// every snapshot. And we will always receive from the controller a complete
+// list of SnapshotConfig for all snapshots that the controller knows about.
+// Then after the snapshot with uuid=2 has been created, we can also get
+// a SnapshotConfig for uuid=1, where the counter in RollbackCmd will be +1.
+// Thus, we will roll back to the state of the logical volume at the time
+// of the snapshot with uuid=1, and the snapshot with uuid=2 will be marked
+// in the info message how IMPLICITLY_DELETED (If the controller did not take
+// care of deleting the snapshot with uuid=2 before the rollback command).
+message RollbackCmd {
+  // UUID of the snapshot to rollback to. Is the snapshot_uuid
+  // from the SnapshotConfig and the real and unique name of the
+  // snapshot in ZFS. This is the rollback target for volume_uuid.
+  string snapshot_uuid = 1;
+  string volume_uuid = 2;    // UUID of the volume to rollback
```

Comment:
Do we need `snapshot_uuid` and `volume_uuid` here? As I can see, `RollbackCmd` included into `SnapshotConfig`. `SnapshotConfig` has both fields inside. What is the goal to duplicate fields?

Reply:
Seems it was in comment from @eriknordmark https://github.com/lf-edge/eve/pull/2633#discussion_r880728729. Do we plan to reuse RollbackCmd somewhere else?